app/src/main/java/me/hufman/androidautoidrive/phoneui/MusicPlayerActivity.kt

```diff
@@ -85,9 +106,18 @@ class MusicPlayerActivity: AppCompatActivity() {
 }
```

Comment:
Add a clause to `onBackPressed` to go back to the NowPlaying fragment from the queue view

Reply:
Added clause in `onBackPressed()`